code/modules/modular_computers/computers/item/pda/pda_presets.dm

```diff
@@ -1,117 +1,211 @@
+/obj/item/modular_computer/tablet/pda/preset/network_admin
+	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
+	greyscale_colors = "#EEDC43#69DBF3#00CC00"
+
+/obj/item/modular_computer/tablet/pda/preset/janitor
+	greyscale_colors = "#933ea8#235AB2"
+
+/obj/item/modular_computer/tablet/pda/preset/chaplain
+	greyscale_config = /datum/greyscale_config/tablet/chaplain
+	greyscale_colors = "#333333#D11818"
+
+/obj/item/modular_computer/tablet/pda/preset/botanist
+	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
+	greyscale_colors = "#50E193#E26F41#71A7CA"
+
+/obj/item/modular_computer/tablet/pda/preset/cook
+	greyscale_colors = "#FAFAFA#A92323"
+
+/obj/item/modular_computer/tablet/pda/preset/artist
+	greyscale_colors = "#3E1111#112334"
 
-/obj/item/modular_computer/tablet/pda/preset/basic/fountainpen // QM, Lawyer, Curator, Bartender
+/obj/item/modular_computer/tablet/pda/preset/fountainpen //Lawyer, Curator, Bartender
 	pen_type = /obj/item/pen/fountain
 
+/obj/item/modular_computer/tablet/pda/preset/fountainpen/lawyer
+	greyscale_colors = "#4C76C8#FFE243"
+
+/obj/item/modular_computer/tablet/pda/preset/fountainpen/bartender
+	greyscale_colors = "#333333#C7C7C7"
+
+/obj/item/modular_computer/tablet/pda/preset/fountainpen/curator
+	desc = "A small experimental microcomputer."
+	icon_state = "pda-library"
+	icon_state_unpowered = "pda-library"
+	icon_state_powered = "pda-library"
+	greyscale_config = null
+	greyscale_colors = null
+
 // Honk
-/obj/item/modular_computer/tablet/pda/preset/basic/clown
+
+/obj/item/modular_computer/tablet/pda/preset/mime
+	greyscale_config = /datum/greyscale_config/tablet/mime
+	greyscale_colors = "#FAFAFA#EA3232"
+	pen_type = /obj/item/toy/crayon/mime
+
+/obj/item/modular_computer/tablet/pda/preset/clown
 	desc = "A hilarious PDA often given to station pranksters."
-	finish_color = "pink"
+	icon_state = "pda-clown"
+	icon_state_unpowered = "pda-clown"
+	icon_state_powered = "pda-clown"
+	greyscale_config = /datum/greyscale_config/tablet/clown //config doesn't actually do anything but -
+	greyscale_colors = null //- hey if you want to set up some cool shit go ahead
 	pen_type = /obj/item/toy/crayon/rainbow
 
-/obj/item/modular_computer/tablet/pda/preset/basic/clown/Initialize(mapload)
+/obj/item/modular_computer/tablet/pda/preset/clown/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/slippery, 120, NO_SLIP_WHEN_WALKING)
 
+// Honk end
+
+/obj/item/modular_computer/tablet/pda/preset/security
+	greyscale_colors = "#EA3232#0000cc"
+
+/obj/item/modular_computer/tablet/pda/preset/security/detective
+	greyscale_colors = "#805A2F#990202"
+
+/obj/item/modular_computer/tablet/pda/preset/security/warden
+	greyscale_config = /datum/greyscale_config/tablet/stripe_split
+	greyscale_colors = "#EA3232#0000CC#363636"
+
+/obj/item/modular_computer/tablet/pda/preset/security/warden/Initialize(mapload)
+	starting_files |= list(
+		new /datum/computer_file/program/secureye,
```

Comment:
Why are you giving them secureye here? Where was this before?

Reply:
it was, original one has secureye, it is just repathed so it may seem weird but it was there before